src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * The TaskList contains the list of tasks and operations to modify the list.
+ */
+public class TaskList {
+    private ArrayList<Task> taskList;
```

Comment:
Perhaps it would be more appropriate to name it tasks instead of taskList?

Reply:
Alright thanks